University-advisor/Forms/MainForm.cs

```diff
@@ -15,9 +15,12 @@ namespace University_advisor.Forms
     public partial class MainForm : Form
     {
         string currentUser;
+        ArrayList statusList = new ArrayList() { "Student", "Graduate", "Lecturer" };
+
         public MainForm(string username)
         {
             InitializeComponent();
+            SetValues();
```

Comment:
Comment is not related to this line however, you have to change things in this lines:
25) Give it more meaningful name.
44, 79) Delete useless methods.
137) Use implicit declaration.

Reply:
Fixed.